### DISCUSSION TITLE
Your homepage nests a <button> inside an <a>

### DISCUSSION THREAD

**Discussion by marcelhageman** [ant-design/ant-design#43068] [General]
Found on: https://ant.design/

See the two big links that look like buttons. This is invalid HTML, with two conflicting interactive elements nested inside one another.

If you want a link to look like a button, that's fine, but you're supposed to use CSS for that.

```HTML
<a href="/components/overview/">
  <button type="button" class="ant-btn css-12jzuas ant-btn-primary ant-btn-lg">
    <span>Getting Started</span>
  </button>
</a>
```

That could and should simply be:

```HTML
<a href="/components/overview/">
  Getting Started
</a>
```

You never nest a button inside an anchor or the other way around. HTML standards matter for many reasons (a11y, keyboard navigation, potential awkward bugs, etc.) and your great library should be a leading example of how to get it right.

**Comment by marcelhageman** [comment]
More than a year later, your homepage still makes your project look like amateur-hour.

**Reply by afc163** [reply]
@marcelhageman Fixing in https://github.com/ant-design/ant-design/pull/49848 now! Thanks for letting me know, and sorry for the delayed response!
